report_thesis/src/sections/pyhat_contribution.tex

```diff
@@ -0,0 +1,22 @@
+\section{PyHAT Contribution}\label{sec:pyhat_contribution}
+As part of our research, we have made several contributions to \gls{pyhat}, which offers a user-friendly interface designed for performing machine learning and data analysis tasks specifically for hyperspectral data.
+This collaboration was initiated through a series of discussions with two members \gls{usgs} responsible for managing \gls{pyhat}, wherein we identified mutual challenges and opportunities for integrating our solutions into the library.
```

Comment:
Generally wanted to rephrase a bit here. And it isn't a library entirely. More like a tool that has a library for core functionality, including an application 'layer'.
```suggestion
Our collaboration was initiated through a series of discussions with two members from \gls{usgs} that are responsible for \gls{pyhat}, wherein we identified mutual challenges and opportunities for integrating our solutions into the tool.
```